python/vTestInterface.py

```diff
@@ -252,62 +251,32 @@ def getTestDataVCAST(enviroPath):
     return testList
 
 
-def printCoverageListing(enviroPath):
-    """
-    This is used for testing only ...
-    It will print out the coverage for each file in the environment.
-
-    The caller will ensure that the source file is part of the environment
-    The covered_char is used as follows:
-        " " UNCOVERED
-        "*" COVERED
-        "A" ANNOTATED
-        "P" PARTIAL
-        "a" ANNOTATED_PARTIAL
-        "X" NOT_APPLICABLE
-    """
-    splitter = "-" * 80
-    line_num_width = 6
-
-    capi = CoverApi(enviroPath)
-
-    sourceObjects = capi.SourceFile.all()
-    for sourceObject in sourceObjects:
-        sys.stdout.write("=" * 100 + "\n")
-        for line in sourceObject.iterate_coverage():
-            sys.stdout.write(str(line.line_number).ljust(line_num_width))
-            sys.stdout.write(line._cov_line.covered_char() + " | " + line.text + "\n")
-    capi.close()
-
-
 def getUnitData(enviroPath):
     """
     This function will return info about the units in an environment
     """
     unitList = list()
     try:
         # this can throw an error of the coverDB is too old!
-        capi = CoverApi(enviroPath)
+        api = UnitTestApi(enviroPath)
     except Exception as err:
         print(err)
         raise UsageError()
 
-    # For testing/debugging
-    # printCoverageListing (enviroPath)
-
-    sourceObjects = capi.SourceFile.all()
+    sourceObjects = api.SourceFile.all()
     for sourceObject in sourceObjects:
-        sourcePath = sourceObject.display_path
-        covered, uncovered, checksum = getCoverageData(sourceObject)
-        unitInfo = dict()
-        unitInfo["path"] = sourcePath
-        unitInfo["functionList"] = getFunctionData(sourceObject)
-        unitInfo["cmcChecksum"] = checksum
-        unitInfo["covered"] = covered
-        unitInfo["uncovered"] = uncovered
-        unitList.append(unitInfo)
-
-    capi.close()
+        if sourceObject.is_instrumented:
```

Comment:
Did `capi.SourceFile` previously only give instrumented files?

I am unsure of the implication of this change if, e.g., you have an environment with `COVERAGE: NONE` -- does that affect `unitInfo["functionList"] = getFunctionData(sourceObject)`?